test/unit/loops.spec.ts

```diff
@@ -515,13 +515,16 @@ for (const testCase of [
     "for (const a in {}) { continue; }",
     "for (const a of []) { continue; }",
 ]) {
+    const expectContinueVariable: util.TapCallback = builder =>
+        expect(builder.getMainLuaCodeChunk()).toMatch("local __continue2");
+
     const expectContinueGotoLabel: util.TapCallback = builder =>
         expect(builder.getMainLuaCodeChunk()).toMatch("::__continue2::");
 
     util.testEachVersion(`loop continue (${testCase})`, () => util.testModule(testCase), {
-        [tstl.LuaTarget.Universal]: builder => builder.expectDiagnosticsToMatchSnapshot([unsupportedForTarget.code]),
-        [tstl.LuaTarget.Lua50]: builder => builder.expectDiagnosticsToMatchSnapshot([unsupportedForTarget.code]),
-        [tstl.LuaTarget.Lua51]: builder => builder.expectDiagnosticsToMatchSnapshot([unsupportedForTarget.code]),
+        [tstl.LuaTarget.Universal]: expectContinueVariable,
+        [tstl.LuaTarget.Lua50]: expectContinueVariable,
+        [tstl.LuaTarget.Lua51]: expectContinueVariable,
```

Comment:
I would expect at least one test that asserts that _this actually works_. For example it might be a good idea to do a `testEachVersion` for the "for with continue" and "while with continue" tests

Reply:
Good idea. I've modified all of the `continue` tests to test against all versions.